README.md

```diff
@@ -5,6 +5,13 @@ Node.js API (N-API), along with library code that enables
 backward-compatibility with use with older versions of Node.js that do
 not have N-API built-in.
 
+### API Documentation
+
+ - [ABI-Stable C APIs in Node.js](https://nodejs.org/dist/latest-v7.x/docs/api/n-api.html)
```

Comment:
Yes. I wanted to link to https://github.com/nodejs/node/blob/master/doc/api/n-api.md for now, but links to external markdown files confuse the Doxygen generator.